packages/react-storage/src/components/StorageBrowser/context/actions/listLocationItemsAction.ts

```diff
@@ -106,6 +107,7 @@ export async function listLocationItemsAction(
   };
 
   const output = await list(listInput);
+  checkRequiredKeys(output, `ListPaginateWithPathOutput`, ['items']);
```

Comment:
What can we say about each item?

Reply:
updated to check individual item's schema